docs/install/install.rst

```diff
@@ -102,11 +111,11 @@ proceed with the installation, you will need to activate the environment:
 
 Cloning POSYDON
 ---------------
-Clone the repository in a local directory, e.g. `/home/POSYDON/`, with
+Clone the repository in a local directory, e.g. ``/home/POSYDON/``, with
 
 .. code-block::
 
-    git clone https://github.com/POSYDON-code/POSYDON.git
+    git clone https://github.com/POSYDON-code/POSYDON.git --branch v1.0.4
```

Comment:
Are we going to have a branch `v1.0.4` instead of a specified commit on `main`?

Reply:
This also works with specific tags/releases :) 
https://stackoverflow.com/questions/45241502/how-can-i-git-clone-a-specific-release